geometry/geometry_tests/screen_test.cpp

```diff
@@ -50,6 +50,54 @@ UNIT_TEST(ScreenBase_P2G2P)
   TEST(is_equal(pg, screen.PtoG(pp)), ());
 }
 
+UNIT_TEST(ScreenBase_3dTransform)
+{
+  ScreenBase screen;
+
+  double const rotationAngle = math::pi4;
+
+  screen.SetFromRects(m2::AnyRectD(m2::RectD(50, 25, 55, 30)), m2::RectD(0, 0, 200, 400));
+  screen.ApplyPerspective(rotationAngle, rotationAngle, math::pi / 3.0);
+
+  TEST(screen.PixelRectIn3d().SizeX() < screen.PixelRect().SizeX(), ());
+  TEST(screen.PixelRectIn3d().SizeY() < screen.PixelRect().SizeY(), ());
+
+  m2::PointD pp(screen.PixelRect().SizeX() / 2.0, screen.PixelRect().SizeY());
+  m2::PointD p3d = screen.PtoP3d(pp);
+  TEST(p3d.EqualDxDy(m2::PointD(screen.PixelRectIn3d().SizeX() / 2.0, screen.PixelRectIn3d().SizeY()), 1.0e-3), ());
```

Comment:
Please, create a named constant for 1.0e -3 , like:
```c++
constexpr double kEps = 1.0e-3;
...
```
and use it instead of 1.0-3 or 0.001.